app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::ReviewsController < ApplicationController
+  def index
+    render json: Review.where(coffeeshop_id: params["coffeeshop_id"])
```

Comment:
Piggy-backing on Nicks comment, is this endpoint for ALL reviews being used? We should be grabbing all associated reviews when we fetch for the coffee shop, either through the association or a serializer